src/examples/component-definition/xml/example-component-definition.xml

```diff
@@ -46,17 +49,52 @@
         source="https://github.com/usnistgov/oscal-content/blob/master/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_MODERATE-baseline_profile.xml">
       <description>
         <p>MongoDB control implementations for NIST SP 800-53 revision 5.</p>
-      </description>
+      </description>          
+      <implemented-requirement
+          uuid="cf8338c5-fb6e-4593-a4a8-b3c4946ee080"
+          control-id="sc-8">
+        <description>
+          <p>MongoDB's implementation of SC-8 control. The details of the implementation are provided at the statement level.</p>
+        </description>
+        <set-parameter param-id="sc-8_prm_1">
+            <value>confidentiality</value>
+        </set-parameter>
+        <statement 
+          statement-id="sc-8_smt" 
+          uuid="bb9219b1-e51c-4680-abb0-616a43bbfbb1">      
+            <description>
+              <p>MongoDB implements TLS 1.x to protect the 
+                <insert type="param" id-ref="sc-8_prm_1"/> 
+              of transmitted data by encrypting data in transit,
+              preventing unauthorized disclosure or changes to information
+              during transmission.</p>
+            </description>
+        </statement>
+      </implemented-requirement>
       <implemented-requirement
-          uuid="cf8338c5-fb6e-4593-a4a8-b3c4946ee2a0"
-          control-id="sc-8.1">
+        uuid="cf8338c5-fb6e-4593-a4a8-b3c4946ee081"
+        control-id="sc-8.1">
         <description>
-          <p>MongoDB supports TLS 1.x to encrypt data in transit,
-          preventing unauthorized disclosure or changes to information
-          during transmission. To implement TLS, set the PEMKeyFile
-          option in the configuration /etc/mongod.conf to the
-          certificate file's path and restart the the component.</p>
+          <p>MongoDB implements cryptographic mechanisms (TLS 1.x) 
+            to provide cryptographic protection for data in transit.</p>
         </description>
+        <set-parameter param-id="sc-8.1_prm_1">
+            <value>prevent unauthorized disclosure of information</value>
+        </set-parameter>
+        <statement 
+          statement-id="sc-8.1_smt" 
+          uuid="bb9219b1-e51c-4680-abb0-616a43bbfbb1">         
+            <description>
+              <p>To implement cryptographic mechanisms (aka enable TLS 1.x) 
+                to <insert type="param" id-ref="sc-8.1_prm_1"/>, 
+                customers need to set the PEMKeyFile
+                option in the configuration /etc/mongod.conf to the
+                certificate file's path and restart the component.</p>
```

Comment:
Non-blocking, minor: similar whitespace/wordwrapping recommendation. Additionally, it might be a nice stylistic approach to wrap the variable name and file path for that variable in `<code/>` blocks for clarity. I leave that up to you. 

```suggestion
              <p>To implement cryptographic mechanisms (aka enable TLS 1.x) to <insert type="param" id-ref="sc-8.1_prm_1"/>, customers need to set the <code>PEMKeyFile</code> option in the configuration  file <code>/etc/mongod.conf</code> to the certificate file's path and restart the component.</p>
```